test/tools/provisioner-cli/main.go

```diff
@@ -32,12 +32,12 @@ func initLogger() {
 }
 
 // export LOG_LEVEL="trace|debug"
-// export CLOUD_PROVIDER="ibmcloud"
+// export CLOUD_PROVIDER="ibmcloud|azure|libvirt"
 // export TEST_E2E_PROVISION_FILE="/root/provision_ibmcloud.properties"
 // export TEST_E2E_PODVM_IMAGE="/root/e2e-test-image-amd64-20230308.qcow2
 // export TEST_E2E_PROVISION="yes"
 // cd test/e2e
-// ../../cluster-provisioner -action=provision | deprovision | uploadimage
+// ./cluster-provisioner -action=provision | deprovision | uploadimage
```

Comment:
I think this has changed to `./caa-provisioner-cli` now?

Reply:
Yes, I'll fix it, thank you @stevenhorsman